colossalai/inference/modeling/policy/tp_nopadding_llama.py

```diff
@@ -0,0 +1,116 @@
+from functools import partial
```

Comment:
It's unnecessary to split llama modeling into two policies. The purpose of policy is to indicate replaced modules/functions/attrs and to manage models under different parallelism settings, and so that a file `tp_nopadding_llama.py` is sort of weird.